### ISSUE TITLE
the cheat command not working

### ISSUE THREAD

**Issue by Tomperyeeter** [RealMangorage/time-in-a-bottle#13]
Hello, i downloaded this mod and on singleplayer the /tiab command works just fine but when im on my server it does not work at all. Weird tho that yesterday it worked one time i could add some time to the bottle. But after like probably a restart or something the command does not work. and i am /op in my server ive tried it in gamemode 1 as well 

Im using it on 1.20.1 forge version is 47.2.0

**Comment by Shugabush** [comment]
Yes I've had this problem too. The /tiab addTime command doesn't work after you do a /reload.
